### ISSUE TITLE
[dataio] dataio-pyshovel search inside detail textbox (Trac #1573)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1565]
Wouldn't it be great if we could search through that long text box for a specific thing?

Proposed: `/` and `?` searching like in `less`. Let's start off with a basic text search before getting fancier.

Migrated from https://code.icecube.wisc.edu/ticket/1573
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:42", 
    "description": "Wouldn't it be great if we could search through that long text box for a specific thing?\n\nProposed: `/` and `?` searching like in `less`. Let's start off with a basic text search before getting fancier.", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067102580394", 
    "component": "combo core", 
    "summary": "[dataio] dataio-pyshovel search inside detail textbox", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2016-03-01T21:42:41", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160302T17:56:18: kjmeagher commented:

> while you are at it, `Home` and `End` should go the first and last event respectively in the file.

**Comment by icecube-trac** [comment]
Trac update at 20160302T23:49:11: 
* david.schultz changed resolution from "" to "fixed"
* david.schultz commented:

> In changeset 2442:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2442"
> searching in the shovel. closes #1565. closes #1566
> ```


* david.schultz changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190111T21:51:34: nwhitehorn commented:

> In changeset 169187:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="169187"
> searching in the shovel. closes #1565. closes #1566
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:42: 
* olivas changed milestone from "April 2016" to ""
* olivas commented:

> Milestone April 2016 deleted
